config/routes.rb

```diff
@@ -1,29 +1,36 @@
 Rails.application.routes.draw do
   filter :locale
 
-  root 'home#index'
+  root 'main#index'
 
-  resources :cards
-  resources :user_sessions, only: [:new, :create, :destroy]
-  resources :users, only: [:new, :create, :destroy]
+  scope module: 'home' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.